rollbar/__init__.py

```diff
@@ -276,7 +276,7 @@ def _get_pylons_request():
 
 ## public api
 
-def init(access_token, environment='production', **kw):
+def init(access_token, environment='production', scrub_fields=list(), **kw):
```

Comment:
This is potentially dangerous.
It's discouraged to define default argument values with mutable objects.

http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments

Reply:
Yeah, you are right!

Reply:
This should be fine. If a literal list object was used `[]` than you would have a mutable object. `list()` is not an object.

Reply:
Also hey gang 👋

Reply:
It isn't an issue here because the default isn't used instead it is just a falsey sentinel, but yeah a `None` would work just as well.

Reply:
@rokob 
You're right, it isn't used now.
But who knows what will be in future.
Can you guarantee next contributor won't overlook that?